sshkey-attest/src/proto.rs

```diff
@@ -0,0 +1,27 @@
+//! Serialisable formats of attested ssh keys
+
+use serde::{Deserialize, Serialize};
+use sshkeys::PublicKey;
+use webauthn_rs_core::attestation::AttestationFormat;
+use webauthn_rs_core::proto::{ParsedAttestation, RegisteredExtensions};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+/*
+#[serde(
+    try_from = "SerialisableAttestedPublicKey",
+    into = "SerialisableAttestedPublicKey"
+)]
+*/
```

Comment:
Is there any reason to serialise this struct? Can this commented attribute can be deleted?